### PR TITLE
Translate `../../library/string.rst:420-430`

### PR DESCRIPTION
**Pull request by heyimkaijoe**
translate `../../library/string.rst:420` to `../../library/string.rst:430`

### CODE REVIEW

File: library/string.po [hunk 1]
```diff
@@ -545,6 +545,9 @@ msgid ""
 "``'X'``, underscores will be inserted every 4 digits.  For other "
 "presentation types, specifying this option is an error."
 msgstr ""
+"``'_'`` 選項表示對於浮點表示類型和整數表示類型``'d'`` 使用下劃線作為千位分隔"
+"符號。對於整數表示類型``'b'``，``'o'``，``'x'``和``'X'``，每4位數字會插入底"
```
Comment (ken71301) [inline library/string.po]: 英文與中文間需空格
```suggestion
"符號。對於整數表示類型 ``'b'``，``'o'``，``'x'`` 和 ``'X'``，每4位數字會插入底"
```

**Review by rockleona — APPROVED** [review]
LGTM!

File: library/string.po [hunk 2]
```diff
@@ -556,6 +559,8 @@ msgid ""
 "including any prefixes, separators, and other formatting characters. If not "
 "specified, then the field width will be determined by the content."
 msgstr ""
+"*width* 是一個十進制整數，定義了最小總字段寬度，包括任何前綴、分隔符號和其他"
```
Comment (mattwang44) [inline library/string.po]:
```suggestion
"*width* 是一個十進制整數，定義了最小總欄位寬度，包括任何前綴、分隔符號和其他"
```

File: library/string.po [hunk 3]
```diff
@@ -556,6 +559,8 @@ msgid ""
...
+"格式字元。如果未指定，則字段寬度將由內容決定。"
```
Comment (mattwang44) [inline library/string.po]:
```suggestion
"格式字元。如果未指定，則欄位寬度將由內容決定。"
```

File: library/string.po [hunk 4]
```diff
@@ -545,6 +545,9 @@ msgid ""
 "``'X'``, underscores will be inserted every 4 digits.  For other "
 "presentation types, specifying this option is an error."
 msgstr ""
+"``'_'`` 選項表示對於浮點表示類型和整數表示類型 ``'d'`` 使用下劃線作為千位分隔"
```
Comment (mattwang44) [inline library/string.po]: presentation type 的 type 是類型還是型別呢？

File: library/string.po [hunk 5]
```diff
@@ -545,6 +545,9 @@ msgid ""
...
+"符號。對於整數表示類型 ``'b'``，``'o'``，``'x'`` 和 ``'X'``，每4位數字會插入底"
```
Comment (mattwang44) [inline library/string.po]:
```suggestion
"符號。對於整數表示類型 ``'b'``，``'o'``，``'x'`` 和 ``'X'``，每 4 位數字會插入底"
```

**Comment by mattwang44** [comment]
rebase

**Review by ken71301 — COMMENTED** [review]
as @mattwang44 's review, there are still some words that need to be fix or reconsider, thanks.
@heyimkaijoe would you fix this pr?

File: library/string.po (hunk 2 above)
Comment (heyimkaijoe) [reply library/string.po]: 已修改

File: library/string.po (hunk 3 above)
Comment (heyimkaijoe) [reply library/string.po]: 已修改

File: library/string.po (hunk 4 above)
Comment (heyimkaijoe) [reply library/string.po]: 應翻為 `型別` 才對
已修改

File: library/string.po (hunk 5 above)
Comment (heyimkaijoe) [reply library/string.po]: 已修改

**Review by ken71301 — APPROVED** [review]
LGTM!

**Pull request merged**